mteb/tasks/Retrieval/en/MedicalQARetrieval.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class MedicalQARetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="MedicalQA",
+        description="The dataset consists 2048 medical question and answer pairs.",
+        reference="https://github.com/lauramanor/legal_summarization",
+        dataset={
+            "path": "mteb/medical_qa",
+            "revision": "ae763399273d8b20506b80cf6f6f9a31a6a2b238",
+        },
+        type="Retrieval",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["en"],
+        main_score="ndcg_at_10",
+        date=None,
```

Comment:
Should be added as a range (start_date, end_date). You can give your best guess if not available.